prdoc/pr_7634.prdoc

```diff
@@ -13,4 +13,4 @@ crates:
   - name: frame-support-procedural
     bump: major
   - name: bridge-hub-westend-runtime
-    bump: none
+    bump: patch
```

Comment:
Why this change?

Reply:
When working on the prdoc for this PR I realized that I also derived `DecodeWithMemTracking` in some structures from `bridge-hub-westend-runtime` in pr #7634. So I guess it should also be `patch`.